DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -81,6 +83,25 @@ bool refactoredTrackingSystemEnabled
             return (candidateNumber, delegateRegistrationModel.Approved);
         }
 
+        public string RegisterDelegateByCentre(DelegateRegistrationModel delegateRegistrationModel)

```

Comment:
Do we want a couple of unit tests to check that the SetPasswordByCandidateNumber method is called when it should be, and not called when it shouldn't, or is that a little too basic?